assets/tailwind.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
     preflight: false
   },
   blocklist: ["container", "collapse"],
-  important: ".body-wrapper",
+  important: "body",
```

Comment:
Didn't we get rid of this field? Why do we need this?

Reply:
It's in the elaborated commit message hehe 
<img width="732" alt="image" src="https://github.com/user-attachments/assets/3cda5496-e636-4eb4-ad97-b2f737a55286" />